### PR TITLE
Fixed a bad migration that caused mysql to act up

### PR DESCRIPTION
**Pull request by mitsuhiko**

### CODE REVIEW

File: src/sentry/south_migrations/0184_auto__del_field_group_checksum__del_unique_group_project_checksum__del.py [hunk 1]
```diff
@@ -12,13 +13,16 @@ def forwards(self, orm):
         db.delete_unique('sentry_groupedmessage', ['project_id', 'checksum'])
 
         # Deleting field 'Group.checksum'
-        db.delete_column('sentry_groupedmessage', 'checksum')
+        if not is_mysql():
+            db.delete_column('sentry_groupedmessage', 'checksum')
 
         # Deleting field 'Event.checksum'
         db.delete_column('sentry_message', 'checksum')
+        pass
```
Comment (mattrobenolt) [inline src/sentry/south_migrations/0184_auto__del_field_group_checksum__del_unique_group_project_checksum__del.py]: gj

File: src/sentry/south_migrations/0184_auto__del_field_group_checksum__del_unique_group_project_checksum__del.py [hunk 2]
```diff
@@ -12,13 +13,16 @@ def forwards(self, orm):
...
 
 
     def backwards(self, orm):
+        return
```
Comment (mattrobenolt) [inline src/sentry/south_migrations/0184_auto__del_field_group_checksum__del_unique_group_project_checksum__del.py]: Nope.

**Comment by mitsuhiko** [comment]
@mattrobenolt fixed. Good?

**Comment by markstory** [comment]
Is this still a necessary/useful change?

**Pull request closed without merging**